.github/workflows/pull-request.yml

```
@@ -0,0 +1,18 @@
+name: Pull Request
+
+on:
+  pull_request:
+    branches:
+    - master
```

```suggestion
      - master
```
Nitpick, consistent spacing for arrays